src/tests.rs

```diff
@@ -17,3 +24,181 @@ fn flatten_unittest() {
     let flat = flatten(&mat);
     assert_eq!(6, flat.len());
 }
+
+#[test]
+fn nss_loopback_simple() {
+    inner_test_simple(ConfigType::NssLoopback);
+}
+
+#[test]
+fn nss_client_vs_boring_server_simple() {
+    inner_test_simple(ConfigType::BsslServer);
+}
+
+#[test]
+fn nss_client_vs_ossl_server_simple() {
+    inner_test_simple(ConfigType::OsslClient);
+}
+
+#[test]
+fn nss_server_vs_boring_client_simple() {
+    inner_test_simple(ConfigType::BsslClient);
+}
+
+#[test]
+fn nss_server_vs_ossl_client_simple() {
+    inner_test_simple(ConfigType::OsslClient);
+}
+
+#[test]
+fn nss_loopback_all_test_cases() {
+    inner_test_all_cases(ConfigType::NssLoopback)
+}
+
+#[test]
+fn nss_server_vs_boring_client_all_test_cases() {
+    inner_test_all_cases(ConfigType::BsslClient)
+}
+
+#[test]
+fn boring_server_vs_nss_client_all_test_cases() {
+    inner_test_all_cases(ConfigType::BsslServer)
+}
+
+#[test]
+fn ossl_server_vs_nss_client_all_test_cases() {
+    inner_test_all_cases(ConfigType::OsslServer)
+}
+
+#[test]
+fn nss_server_vs_ossl_client_all_test_cases() {
+    inner_test_all_cases(ConfigType::OsslClient)
+}
```

Comment:
Please find some way to parametrize these.

Reply:
Opened an issue to address this in the future. 
https://github.com/mozilla/tls-interop/issues/8